src/context.cc

```diff
@@ -263,30 +266,38 @@ void ContextBase::onForeignFunction(uint32_t foreign_function_id, uint32_t data_
 }
 
 FilterStatus ContextBase::onNetworkNewConnection() {
```

Comment:
not sure whether we should call failClose for both `WasmStreamType::Downstream` and `WasmStreamType::Upstream`.. either way, the connection will be closed in the next callback on On{Downstream,Upstream}Data.

Reply:
yeah I think we should do to be aligned with the behavior of http ones.